package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@blackbaud/sky-addin-client",
-  "version": "1.3.0",
+  "version": "1.3.1",
```

Comment:
We can bump up to `1.4.0` for this new feature.

Reply:
Done, thank you!